task_3.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+function tickets(person) {
+  const cashbox = [];
```

Comment:
one thing to consider (no need to change your implementation, just a thought) - this calculation could be simplified to work with counts (numbers). so for example you could have two variables for 25 & 50 counts and change them within the loop. so there would be no need to manipulate the array and check it's length in the end